test/.eslintrc.js

```diff
@@ -15,5 +15,17 @@ module.exports = {
   },
   env: {
     mocha: true
-  }
+  },
+  overrides: [
+    {
+      files: ['*.mjs'],
+      rules: {
+        // TODO: remove this when we decide on how to address
+        // here: https://issues.newrelic.com/browse/NEWRELIC-3321
+        'node/no-unsupported-features/es-syntax': 'off',
+        'node/no-unpublished-import': 'off',
```

Comment:
I will open PR to put this in the shared eslint ruleset. we already do it for the cjs equivalents: https://github.com/newrelic/eslint-config-newrelic/blob/main/index.js#L108